### DISCUSSION TITLE
Feature Request: Excel Export from List View

### DISCUSSION THREAD

**Discussion by gtolarc** [payloadcms/payload#121] [Feature Requests & Ideas]
I wish there was a function button on the top right of the table that can excel export or print the contents of 1 page where the current filter, ordering, and pagination are applied. This is a useful function in actual cms work.

**Comment by jmikrut** [comment]
Hey @gtolarc—this is actually a planned feature and will be deployed within the next two months or so! 

We will be making our roadmap public shortly, and you’ll see this feature set reflected within it!

**Reply by clhome** [reply]
hi，is this feature complete？

**Reply by leikoilja** [reply]
@jmikrut any progress on this? 
i believe https://github.com/payloadcms/payload/discussions/2425 is a duplicate of this

**Comment by webmuch** [comment]
I was able to make my own:

https://github.com/payloadcms/plugin-form-builder/issues/47#issuecomment-1707296877
